### PR TITLE
Improve examples

### PR DESCRIPTION
**Pull request by erikarvstedt**

### CODE REVIEW

**Comment by nixbitcoin** [comment]
On Debian `sysctl` isn't available to non-root users, so `./deploy-container.sh` yields

```
./deploy-container.sh: line 11: sysctl: command not found
```

It might also have to be run with sudo.

File: examples/minimal-configuration.nix [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{ config, pkgs, lib, ... }: {
+  imports = [
+    <nix-bitcoin/modules/nix-bitcoin.nix>
+    <nix-bitcoin/modules/generate-secrets.nix>
```
Comment (nixbitcoin) [inline examples/minimal-configuration.nix:4]: Shouldn't this be `<nix-bitcoin/modules/secrets/generate-secrets.nix>`? I'm getting
```
error: file 'nix-bitcoin/modules/generate-secrets.nix' was not found in the Nix search path (add it using $NIX_PATH or -I), at /home/nix-bitcoin/nix-bitcoin/examples/minimal-configuration.nix:4:5
```

Comment (erikarvstedt) [reply examples/minimal-configuration.nix]: Thanks, this was an untested last-minute change. Fixed.

**Comment by erikarvstedt** [comment]
The `sysctl` issue predates this PR. Fixed in [`deploy-container.sh: run sysctl as root`](https://github.com/fort-nix/nix-bitcoin/pull/250/commits/ce38aed12315358b4f76891bec20802c5e0cbade).

File: README.md [hunk 2]
```diff
@@ -48,8 +48,20 @@ shut down immediately. They leave no traces (outside of `/nix/store`) on the hos
   NixOps can be used to deploy to various other backends like cloud providers.\
   Requires: [Nix](https://nixos.org/nix/), [VirtualBox](https://www.virtualbox.org)
 
+- [`./deploy-container-minimal.sh`](examples/deploy-container-minimal.sh) creates a
+  container defined by [minimal-configuration.nix](examples/minimal-configuration.nix) that
+  doesn't use the [secure-node.nix](modules/presets/secure-node.nix) preset.
+  Also shows how to use nix-bitcoin in an existing NixOS config.\
```
Comment (jonasnick) [inline README.md:54]: In what way exactly does this show how to use nix-bitcoin in an existing NixOS config?

**Review by jonasnick — COMMENTED** [review]
uACK mod nit

File: README.md (hunk 2 above)
Comment (erikarvstedt) [reply README.md:54]: Actually, there's not much to show. I've [extended a comment](https://github.com/fort-nix/nix-bitcoin/compare/c1df4cb37a79ef6480e099a661dffac87d3bde3f..5d3221845b0cccf31fafdfffb72aa9bae6d63770) to make it a bit clearer.

Comment (jonasnick) [reply README.md:54]: Oh I see okay

**Review by jonasnick — APPROVED** [review]
ACK 5d3221845b0cccf31fafdfffb72aa9bae6d63770

File: examples/deploy-container.sh [hunk 3]
```diff
@@ -8,20 +8,35 @@ set -euo pipefail
...
 # script in the interactive shell.
 
-if [[ $(sysctl -n net.ipv4.ip_forward) != 1 ]]; then
+if [[ ! -v IN_NIX_SHELL ]]; then
+    echo "Running script in nix shell env..."
+    cd "${BASH_SOURCE[0]%/*}"
+    exec nix-shell --run "./${BASH_SOURCE[0]##*/} $*"
+fi
+
+if [[ $(sysctl -n net.ipv4.ip_forward || sudo sysctl -n net.ipv4.ip_forward) != 1) ]]; then
```
Comment (nixbitcoin) [inline examples/deploy-container.sh:17]: remove `)` in `!= 1)`

**Review by nixbitcoin — APPROVED** [review]
```
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

ACK b574cb097f27e82fdcfa39b5705855993a600464
-----BEGIN PGP SIGNATURE-----

iQIzBAEBCgAdFiEEV3o0Un8+KoXoD+Fk3RH5rVMIs7oFAl+OwA0ACgkQ3RH5rVMI
s7qEaxAAzHrwuL4n19ncL9nAh1Zg9wC6QYUGWHFJ7O1Lc/SE24c8DdyJpikMbDBe
EuaFiCXDscy2rl60md56HESjRteJJQDnrXCNKL3hGodWbwtzIACYH4N1W8+XoqUI
Sc/QFoE5p16Zr2AOAdwspRVfn6SMsC0cxety47HzpMcxL+LJGtWom5eBcZzF8UCd
t5rMMYLdJ1s6ruZlJpY06FtasDztHbGkbim8zM7SVqVIQftvNAYmzq53q4FGPM27
7+s9KtzdAGF3EmGJRL+wgxLvydzlCgoscYJzaOEPym2CK7jdtpB4qftxXTCrKbeR
RfNeGBzGk9EC+QX5ak4VyQrdz9e1VtxecrMH0ekxhFfxsNNlhJK0OZMuzQSO3p8F
xQXUG+LC4DEZB9W538tYn4BcxYuIGDuRo6kO1OWWv4cUyMdWlj3a4sBIE1x3fZ9z
siOvMJAVt20CvP9ttVtgKHRxiLSntKLLdtvHzztshOebDyRs4ANXGThy+7/QwK9M
M8aDmb2CvXQBupzQldY1unmPM3I+ETBjGbEQLEqHzlH4ZDb69A3aYdQwgMNySguN
LbrHw7SRgXRBT8fUi1jDIWz7D1ATdJlZKxsr0f503GdYk7+m+fvFGpfeRxwrcD7l
r0eA8aMpS23DQu/heHxIkk+/SGJBRYc6dZiYiBHWC5J3aigk9x0=
=ZTyn
-----END PGP SIGNATURE-----
```

**Pull request merged**